config/locales/en.yml

```diff
@@ -93,18 +93,18 @@ en:
           <a href='%{privacy_url}'>Privacy Policy</a>
       is_private: "Check to keep your profile private."
       account_information: "Account Information"
-      display_name: "Username:"
+      display_name: "Membername:"
```

Comment:
That works for me. Oh damn, looks like I broke some tests. I'll try and fix this up within the hour